ssl/s3_enc.c

```diff
@@ -273,20 +273,56 @@ int ssl3_finish_mac(SSL_CONNECTION *s, const unsigned char *buf, size_t len, int
          * we calculate the digest when initiating s->s3.handshake_dgst at which
          * point we know what the protocol version is.
          */
+        if (s->negotiated_version == DTLS1_3_VERSION) {
+           /*
+            * In DTLS 1.3 we need to parse the messages that are buffered to
+            * be able to remove message_sequence, fragment_size and fragment_offset
+            * from the Transcript Hash calculation.
+            */
```

Comment:
The indentation here is off by one space.

Reply:
Ah yes I misunderstood the error report. It pointed to the last line but in fact it was the full block. Thanks.